src/core/loss_detection.c

```diff
@@ -408,6 +409,7 @@ QuicLossDetectionOnPacketSent(
             "Sent packet metadata",
             SIZEOF_QUIC_SENT_PACKET_METADATA(TempSentPacket->FrameCount));
         QuicLossDetectionRetransmitFrames(LossDetection, TempSentPacket, FALSE);
+        QuicLossDetectionOnPacketDiscarded(LossDetection, TempSentPacket, FALSE, FALSE);
```

Comment:
We talked in person about a slightly different way to do this. Please update. Thanks!